about.html

```diff
@@ -0,0 +1,163 @@
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care, gain insights on more
+          efficient management processes and become aware of new technology and
+          techniques, which will ultimately help them and their companies: Sell
+          more products, Reduce costs, and Increase efficiencies
+        </p>
+        <p class="about-venue">
+          Please contact us via email for any further question and
+          clarifications about the ACPN conference 2023
+        </p>
+        <a class="email" href="#">info@acpn2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="ending">
+      <div class="holder">
+        <h2>ACPN Conference 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ACPN Conference 2023 will be decided through the
+          annual ACPN logo competition on June 20, 2023.
+        </p>
+        <h4 class="acpn">
+          ACPN NATIONAL <br />
+          CONFERENCE 2023.
+        </h4>
+      </div>
+      <hr class="dash" />
+      <div class="holder">
+        <h2>See the past ACPN Conference</h2>
+        <hr />
+        <p>
+          Take a look at the last two ACPN Conference which took place in Abuja,
+          Nigeria.
+        </p>
+
+        <div id="meeting">
+          <div class="meeting-start">
+            <h3 class="meeting-header">2021</h3>
+            <p class="meeting">ACPN Conference 2021 in Abuja</p>
+          </div>
+
+          <div class="meeting-middle">
+            <h3 class="meeting-header">2019</h3>
+            <p class="meeting">ACPN Conference 2021 in Enugu.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- SUB-SECTION ENDS -->
+
+    <!-- PARTNER STARTS -->
+    <section class="partner sponsor" id="sponsor">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="logos logo-home">
+          <p class="sponsor1">daumkakao</p>
+          <img src="images/coke.png" alt="" />
+          <img src="images/facebook.png" alt="" />
+          <img src="images/amazon.png" alt="" />
+          <img src="images/mozilla.png" alt="" />
+          <img src="images/google.svg" alt="" />
+        </div>
+      </div>
+    </section>
+    <!-- PARTNER ENDS -->
+
+    <!-- FOOTER STARTS -->
+
+    <footer id="footer-home">
+      <div class="footer-content container">
+        <h5>ACPN Conference.</h5>
+        <p class="footer-text">
+          2023 Association of Community Pharmacists of Nigeria <br />
+          All Rights Reserved.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] Nice one making the changes based on spacing, borders, shadows, and paddings as in the original design and as suggested by the previous reviewer. From the last review, I can see that this section follows the right background image color on the design on the `about`, but you've changed it to white. Kindly fix the color back to stick to the design template given 👍 

## Last review given with the right color:
![image](https://user-images.githubusercontent.com/50990925/234150213-d9aeb529-4b60-4435-81f8-77bc7ece500c.png)

## Your present design:
![image](https://user-images.githubusercontent.com/50990925/234150266-9f523106-8561-4e69-8549-9ae77ff05cff.png)

## Design given:
![image](https://user-images.githubusercontent.com/50990925/234150325-4eb5cb11-2f12-48bd-84b0-fc03605bb2d9.png)